Fulfillment/Integration/02_ManagePackagingTypes.md

```diff
@@ -0,0 +1,206 @@
+
+10. Enter the applicable packaging type identifier in the *Packaging type identifier* field. Both letters and numbers can be entered.
+
+11. Click the *Ship-to country* drop-down list and select the country where the shipment is to be sent to.
+
+12. Enter the applicable priority value in the *Priority* field. This field is mandatory.
+    > [Info] The priority value is used by the system to determine the packaging type to be used for a shipment when the criteria of two or more packaging types match. The priority value becomes then the decisive criterion. In the priority scale, the lower the number, the higher the priority, that is, priority 1 is higher than priority 4.
+
+13. Enter the applicable information in the *Additional services (shipping method)* field. Both letters and numbers can be entered.
+
+14. Click the [SAVE] button in the upper right corner.  
+    The *Submitting data...* notice is displayed while saving followed by the *Creation successful* pop-up window. 
+
+     ![Packaging type created](../../Assets/Screenshots/Fulfillment/Settings/PackagingTypes/CreationSuccessful.png "[Packaging type created]")
+
+     The *Create packaging type* view is closed and the *List of packaging types* is displayed again.
+
+15. If necessary, click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button in the upper right corner to display the new packaging type in the list.   
+  
+
+## Edit a packaging type
+
+Edit a packaging type to update any set criteria.
+
+#### Prerequisites
+
+A packaging type has been created, see [Create a packaging type](#create-a-packaging-type).
+
+#### Procedure
+
+*Fulfillment > Settings > Tab PACKAGING TYPES*
+
+![Packaging types](../../Assets/Screenshots/Fulfillment/Settings/PackagingTypes/ListPackagingTypes.png "[Packaging types]")
+
+1. Click the packaging type to be edited. Alternatively, click the checkbox of the packaging type to be edited and the [EDIT] button in the editing toolbar.  
+    The *Edit packaging type* view is displayed.
+
+2. Edit the desired criteria of the packaging type in the corresponding fields.
+
+3. Click the [SAVE] button.   
+  The *Submitting data...* notice is displayed while saving followed by the *Update successful* pop-up window. 
+
+    ![Packaging type updated](../../Assets/Screenshots/Fulfillment/Settings/PackagingTypes/UpdateSuccessful.png "[Packaging type updated]")
+
+    The *Edit packaging type* view is closed and the *List of packaging types* is displayed again.
+
+4. If necessary, click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button in the upper right corner to display the changes in the list.   
+  
+
+## Delete a packaging type
+
+Delete a packaging type that is no longer applicable. To avoid potential issues with existing shipments, it is highly recommended to deactivate a packaging type temporarily instead of deleting it permanently, see [Deactivate a packaging type](#deactivate-a-packaging-type).
```

Comment:
```suggestion
Delete a packaging type that is no longer applicable. To avoid issues with existing shipments, it is highly recommended to deactivate a packaging type instead of deleting it, see [Deactivate a packaging type](#deactivate-a-packaging-type).
```